commons/helpers/pods_helper.py

```diff
@@ -206,21 +229,33 @@ def get_deploy_replicaset(self, pod_name):
                       LogicalNode.get_deploy_replicaset.__name__, error)
             return False, error
 
-    def get_num_replicas(self, replicaset):
+    def get_num_replicas(self, set_type, set_name):
         """
         Helper function to get number of desired, current and ready replicas for given replica set
-        :param replicaset: Name of the replica set
+        :param set_type: Type of the set (replica set or statefulset)
+        :param set_name: Name of the set
         :return: Bool, str, str, str (Status, Desired replicas, Current replicas, Ready replicas)
```

Comment:
so in one case we have 4 o/p going out and in another 3?

Reply:
yes, the outputs of get rs and get sts return different set of values.